tests/kernel/smp/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config TRACE_SCHED_IPI
+	bool "Enable Test IPI"
+	default n
+	depends on SMP
```

Comment:
You don't actually need this, nothing breaks if you enable this when SMP=n, it just does nothing.   But if you want to be pedantically correct: you want to predicate this on SCHED_IPI_SUPPORTED and MP_NUM_CPUS>1, not SMP.  The IPI API is a lower level thing than SMP.

Reply:
Oh, understand. Got it fixed.